api/src/flows.ts

```diff
@@ -235,6 +240,7 @@ class FlowManager {
 				flow.options['return'] = flow.options['return'] ?? '$last';
 
 				this.webhookFlowHandlers[`${method}-${flow.id}`] = handler;
+				this.webhookFlowPreventCaches[`${method}-${flow.id}`] = !!flow.options['preventCache'];
```

Comment:
Currently the app does only show the option when it's a GET webhook, but I believe we'll need to check it in the api/flows side as well, specifically for this pretty rare edge case of: 


1. Create a GET webhook trigger with `preventCache` enabled, so now `flow.options['preventCache']` is true, then save.
2. Update the same webhook trigger from GET to POST. Although the option itself disappears, `flow.options['preventCache']` still has value in the db, and this will still evaluate as true.

(Though understandably "lingering" options is technically something present even for display options and interface options if I'm not mistaken, which leads to why I recall such edge case here)